biome.jsonc

```diff
@@ -57,33 +46,39 @@
           }
         },
         "useArrayLiterals": "error",
-        "noUndeclaredDependencies": "off", // TODO: Need to see why this rule is not detecting monorepo packages
         "noUndeclaredVariables": "error"
       },
-      "performance": {
-        "noAccumulatingSpread": "off",
-        "noDelete": "off"
-      },
       "style": {
+        // The code usage looks suspicious so it should be enabled in a separate PR
         "noCommaOperator": "off",
-        "noInferrableTypes": "off",
-        "noNonNullAssertion": "error",
+        // There are a lot of places we mutate params, should be fixed in an independent PR.
         "noParameterAssign": "off",
         "noRestrictedGlobals": {
           "level": "error",
           "options": {
             "deniedGlobals": ["fetch"]
           }
         },
-        "noUnusedTemplateLiteral": "off",
+        // There are a lot of places we mutate params, should be fixed in an independent PR.
         "noUselessElse": "off",
-        "noVar": "error",
-        "useConst": "error",
-        "useEnumInitializers": "off",
+        // We prefer to use `Math.pow` over `**` operator
         "useExponentiationOperator": "off",
+        // In some cases the enums are initialized with values of other enums
+        "useLiteralEnumMembers": "off",
+        // We prefer to have multiple declarations lines
+        "useSingleVarDeclarator": "off",
+        // We use `+` operator for string concatenation a lot
+        "useTemplate": "off",
+        // We use to export types and object without differentiating
         "useExportType": "off",
```

Comment:
worth discussion to see if we want useExportType and useImportType

Reply:
Created an issue to discuss this: https://github.com/ChainSafe/lodestar/issues/7159